test/gas/validateUserOp.t.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {console2} from "forge-std/Test.sol";
+import {SmartWalletTestBase} from "../CoinbaseSmartWallet/SmartWalletTestBase.sol";
+import {CoinbaseSmartWallet} from "../../src/CoinbaseSmartWallet.sol";
+import {CoinbaseSmartWalletFactory} from "../../src/CoinbaseSmartWalletFactory.sol";
+import {UserOperation} from "account-abstraction/interfaces/UserOperation.sol";
+import {WebAuthn} from "webauthn-sol/WebAuthn.sol";
+
+/// forge-config: default.isolate = true
+contract ValidateUserOpTest is SmartWalletTestBase {
+    bytes32 constant TEST_HASH = keccak256("test operation");
+    CoinbaseSmartWalletFactory factory;
+    
+    function setUp() public override {
+        super.setUp();
+        vm.etch(account.entryPoint(), address(new MockEntryPoint()).code);
+        
+        // Deploy factory
+        CoinbaseSmartWallet implementation = new CoinbaseSmartWallet();
+        factory = new CoinbaseSmartWalletFactory(address(implementation));
+    }
+    
+    function test_k1() public {
+        (uint8 v, bytes32 r, bytes32 s) = vm.sign(signerPrivateKey, TEST_HASH);
+        UserOperation memory op;
+        op.signature = abi.encode(CoinbaseSmartWallet.SignatureWrapper(0, abi.encodePacked(r, s, v)));
+        
+        vm.startSnapshotGas("validation_k1");
+        MockEntryPoint(account.entryPoint()).validateUserOp(address(account), op, TEST_HASH, 0);
+        uint256 gasUsed = vm.stopSnapshotGas();
```

Comment:
was actually looking to measure `account.validateUserOp` not `entryPoint.validateUserOp`!